chalice/awsclient.py

```diff
@@ -483,3 +483,29 @@ def _client(self, service_name):
             self._client_cache[service_name] = self._session.create_client(
                 service_name)
         return self._client_cache[service_name]
+
+    def add_permission_for_authorizer(self, rest_api_id, function_arn,
+                                      random_id):
+        # type: (str, str, str) -> None
+        client = self._client('apigateway')
+        authorizers = client.get_authorizers(restApiId=rest_api_id)
```

Comment:
It does not.  I couldn't find in their docs what the page size, but I think this might be a forward compatibility thing in their API.  The max authorizers for an API is 10 (http://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html) so I don't think we'll be affected.  I can add a comment about that though.